src/tests/ftest/server/metadata.py

```diff
@@ -120,6 +120,7 @@ def thread_control(self, threads, operation):
         self.d_log.debug("IOR {0} Threads Finished -----".format(operation))
         return "PASS"
 
+    @skipForTicket("DAOS-1936/DAOS-1946")
```

Comment:
It's only DAOS-1946, right ?

Reply:
> It's only DAOS-1946, right ?

Restored the original skipForTicket line. It is somewhat related to DAOS-1936 that encounters similar instability when -DER_NOSPACE occurs with metadata (whether it happens as a result of container create here, or container snapshot create as is being explored with DAOS-1936).